src/utils/transak/transak.hooks.ts

```diff
@@ -316,3 +321,12 @@ export const useTransak = (apiKey: string, initialConversion = false) => {
     getDisplayAmount,
   };
 };
+
+export const useTransakApiKey = () => {
+  const { data: activeTier } = useActiveTier();
+
+  return useMemo(() => {
```

Comment:
Not sure this is a good idea because it's easy for technical users to manipulate it.

Assuming Transak supports a way to do this, we probably want users to send a request to our API, so that we can check their tier (so we probably need an indexer) and redirect them back to a Transak page/flow in a way where it is not possible to change the API_KEY to a different one.